packages/@uppy/companion-client/src/index.js

```diff
@@ -6,9 +6,11 @@
 const RequestClient = require('./RequestClient')
 const Provider = require('./Provider')
 const Socket = require('./Socket')
+const tokenStorage = require('./token-storage')
 
 module.exports = {
   RequestClient,
   Provider,
-  Socket
+  Socket,
+  tokenStorage
```

Comment:
does this need to be exported? all the auth token handling is done internally in companion-client isn't it?

Reply:
True, I had initially exported it before realising that it will be loaded internally with `Provider.initPlugin`